packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.routing.js

```diff
@@ -39,7 +39,10 @@ export default /* @ngInject */ ($stateProvider) => {
       ) => {
         // option price is only available for servers migrated to agora
         if (ola.isAvailable() && (!ola.isActivated() || !ola.isConfigured())) {
-          return DedicatedServerInterfacesService.getOlaPrice(serverName, server);
+          return DedicatedServerInterfacesService.getOlaPrice(
+            serverName,
+            server,
```

Comment:
It seems like something went wrong as this PR brings changes it shouldn't. Maybe this needs a rebase

Reply:
I can't revert, what ever I try, the formatter redo this every time I try to commit the develop version. Can you help me ?

Reply:
@Steffy29   don't know if you manage to fix that, but is this change related to prettier ? (If so, maybe disable it just before to commit)

Reply:
@JeremyDec We fixed that via a rebase ;) (outdated)